packages/emoji-mart/src/components/Picker/Picker.js

```diff
@@ -1037,7 +1060,7 @@ export default class Picker extends Component {
                 onMouseLeave={() => this.handleSkinMouseOver()}
                 class="option flex flex-grow flex-middle"
               >
-                <span class={`skin-tone skin-tone-${skin}`}></span>
+                {this.renderSkinToneIcon(this.state.skin)}
```

Comment:
This breaks the skin tone menu having a different skin tone per option.

Current:
<img width="163" alt="Screenshot 2022-11-06 at 2 13 11 PM" src="https://user-images.githubusercontent.com/436043/200190336-0246854e-b70b-4ebb-bbf6-240180e76fcd.png">

This branch:
<img width="165" alt="Screenshot 2022-11-06 at 2 14 33 PM" src="https://user-images.githubusercontent.com/436043/200190349-fded5fe5-6c94-4183-9295-f49efbc9332d.png">
<img width="176" alt="Screenshot 2022-11-06 at 2 12 38 PM" src="https://user-images.githubusercontent.com/436043/200190351-41322732-9bcc-4487-8d29-a50fa1f049ea.png">